app/src/main/java/org/pesmypetcare/mypetcare/services/user/UserManagerAdapter.java

```diff
@@ -154,4 +198,43 @@ public void changeUsername(User user, String newUsername) {
             e.printStackTrace();
         }
     }
+
+    @Override
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method length is 19 lines (max allowed is 15).](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5561369)